### ISSUE TITLE
Can't open or uninstall SteamVR

### ISSUE THREAD

**Issue by DiThi** [ValveSoftware/SteamVR-for-Linux#46]
#### Your system information
* Steam client version (build number or date): Apr 19 2017, at 00:37:25 (beta) and Mar 22 2017, at 23:45:04
* Distribution (e.g. Ubuntu): 16.04
* Graphics driver version (run nvidia-settings): 381.09 (beta)
* Gist for SteamVR System Information: Can't open it
* Opted into Steam client beta?: [Yes/No] Yes
* Opted into SteamVR beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
SteamVR fails to open, the "going to launch game" dialog is always open. This happened before so I tried to uninstall and reinstall it. I can't uninstall it, tried both in Steam beta and normal.

I'm going to install Steam from scratch, I'm just filing this issue for the next person that has this problem, in case there's a quicker way. And of course for following the issue itself.

**Comment by DiThi** [comment]
Nevermind, downgrading SteamVR to non beta worked.

**Comment by ChristophHaag** [comment]
Maybe it would be a good idea to open this issue again - after all, downgrading to stable is just a workaround which resolves neither issue - uninstall not working and the perpetual "updating..." dialog.

You will probably encounter this again in the future, but there are simpler workarounds.

For one, you can still start vrmonitor, you just can't do it from the Steam Library GUI:
```
~/.local/share/Steam/ubuntu12_32/steam-runtime/run.sh ~/.local/share/Steam/SteamApps/common/SteamVR/tools/bin/vrmonitor.sh
```
I think it has one drawback: I don't see any of the SteamVR workshop items in the vrdashboard settings anymore (like SteamVR environments). I'm not really sure but I think starting it from a "wrong" working directory doesn't let it find the files for those items anymore.

Second, instead of downgrading and upgrading again, you can "manually uninstall" SteamVR by just deleting its directory:
```
rm -rf ~/.local/share/Steam/SteamApps/common/SteamVR
```
Then in the Steam Library, go into SteamVR's properties and "verify the integrity" of the files so it will redownload everything. Even though this will restore the directory to exactly the same state it was in before deleting it, it should start to work again when started from the Steam Library GUI (I checked by making a local git repository in the SteamVR directory and committing all files, and checked git status after the problem happened - no changes). There is probably something in the "integrity check" that triggers the thing that fixes it.

**Comment by DiThi** [comment]
You're right, I'll leave it open until it's resolved (the non opening part at least).

**Comment by communityus** [comment]
Any progress made on using steam workshop items in vr on linux or using the workshop tools themselves on linux?
